src/main/scala/dpla/ingestion3/enrichments/StringUtils.scala

```diff
@@ -161,5 +162,56 @@ object StringUtils {
       *   whitespace (\s)
       */
     private def beginAndEndPunctuationToRemove = """[;:/,-\\t\\r\\n\s]"""
+
+    /**
+      * Removes matching leading and trailing brackets (square, round and curly braces)
+      * from a string
+      */
+    val stripBrackets: SingleStringEnrichment = {
+      val replacemnetRegex = for( (p, r) <- bracketPatterns if value.matches(p)) yield r
+      // TODO is there a cleaner or safer way to convert Iterator[String] to String than mkString?
```

Comment:
FWIW I do not know of another way to do the conversion.

Reply:
thanks --0 I'll fix it when I rebase